common/init.sls

```diff
@@ -1,11 +1,38 @@
 {% from tpldir ~ '/map.jinja' import config with context %}
 
 {% if grains['kernel'] != 'Darwin' %}
-python-pkgs:
+# Ubuntu has python2 as default python
+python2:
   pkg.installed:
     - pkgs:
-      - python-pip
+      - python
       - python-dev
+
+python3:
+  pkg.installed:
+    - pkgs:
+      - python3
+
+# Ensure pip is default by purging pip3
+pip:
+  pkg.installed:
+    - pkgs:
+      - python-pip
+    - reload_modules: True
+
+pip3:
+  pkg.purged:
+    - pkgs:
+      - python3-pip
+
+# Virtualenv package creates virtualenv and virtualenv-3.4 executables
```

Comment:
In my experience I needed to `pip3 install virtualenv` before I got virtualenv3. Should we make two keys here, one with `bin_env: pip` and one with `bin_env: pip3`?